### ISSUE TITLE
Errno::EACCES at /node/version/view 

### ISSUE THREAD

**Issue by theodotos** [ytti/oxidized#902]
Hi,

I get this when I try to access any of the different versions:

```
Errno::EACCES at /node/version/view
Permission denied @ rb_sysopen - /var/lib/gems/2.3.0/gems/oxidized-web-0.9.1/lib/oxidized/web/views/version.haml

    file: template.rb location: initialize line: 225 
```

Accessing the latest version of the config works fine and so is diff.

My config:

```
---
username: username
password: password
model: junos
interval: 3600
use_syslog: false
debug: false
threads: 30
timeout: 20
retries: 3
prompt: !ruby/regexp /^([\w.@-]+[#>]\s?)$/
rest: 0.0.0.0:8888
next_adds_job: false
vars: {}
groups: {}
models: {}
pid: "/srv/oxidized/.config/oxidized/pid"
input:
  default: ssh, telnet
  debug: false
  ssh:
    secure: false
output:
  default: git
  git:
    user: oxidized
    email: oxidized@example.com
    repo: "/srv/oxidized/.config/oxidized/devices.git"
source:
  default: csv
  csv:
    file: "/srv/oxidized/.config/oxidized/router.db"
    delimiter: !ruby/regexp /:/
    map:
      name: 0
      ip: 1
      model: 2
      username: 3
      password: 4
    vars_map:
      enable: 5
    gpg: false
model_map:
  cisco: ios
  juniper: junos
```

**Comment by theodotos** [comment]
Nailed it:

```
ls -la /var/lib/gems/2.3.0/gems/oxidized-web-0.9.1/lib/oxidized/web/views/version.haml
-rw------- 1 root root 591 Jun 27 15:04 /var/lib/gems/2.3.0/gems/oxidized-web-0.9.1/lib/oxidized/web/views/version.haml
```

The service runs as user *oxidized* and fails to read this file:

I fixed it with:

```
chmod +r /var/lib/gems/2.3.0/gems/oxidized-web-0.9.1/lib/oxidized/web/views/version.haml
```

I guess a bug in the packaging?

**Comment by laf** [comment]
It does look like packaging as the file is listed with the same properties in the github repo as other files.

**Comment by ytti** [comment]
I've released 0.9.2, it should fix the packaging. Sorry about that.
